OCCommunicationLib/OCCommunicationLib/OCWebDavClient/OCWebDAVClient.m

```diff
@@ -654,14 +663,17 @@ - (void) setRedirectionBlockOnDatataskWithOCCommunication: (OCCommunication *) s
             
             if (sharedOCCommunication.isCookiesAvailable) {
                 //We add the cookies of that URL
-                request = [UtilsFramework getRequestWithCookiesByRequest:requestRedirect andOriginalUrlServer:self.originalUrlServer];
+                // really needed? are not automatically added by NSURLSession?
+                requestRedirect = [UtilsFramework getRequestWithCookiesByRequest:requestRedirect andOriginalUrlServer:self.originalUrlServer];
```

Comment:
Right, probably not needed now because we are using NSURLSession on AFNetworking 3 but probably this code was from AFNetworking 2